File: src/main/java/mods/eln/sim/nbt/NbtElectricalGateOutput.java
package mods.eln.sim.nbt;

import mods.eln.Eln;
import mods.eln.Vars;
import mods.eln.misc.Utils;

public class NbtElectricalGateOutput extends NbtElectricalLoad {

    public NbtElectricalGateOutput(String name) {
        super(name);
        Vars.signalCableDescriptor.applyTo(this);
    }

    public String plot(String str) {
        return Utils.plotSignal(getU(), getCurrent()); //return //str + " " + Utils.plotVolt("", getU()) + Utils.plotAmpere("", getCurrent());
    }
}

File: src/main/java/mods/eln/sim/nbt/NbtElectricalGateOutput.kt
package mods.eln.sim.nbt

import mods.eln.Eln
import mods.eln.misc.Utils

class NbtElectricalGateOutput(name: String) : NbtElectricalLoad(name) {

    init {
        Eln.signalCableDescriptor.applyTo(this)
    }

    fun plot(str: String): String {
        return Utils.plotSignal(u, current)
    }
}
